README.md

```diff
@@ -173,6 +174,10 @@ create-or-update-tags \
 --tags ResourceId=my-auto-scaling-group,ResourceType=auto-scaling-group,Key=spot-enabled,Value=true,PropagateAtLaunch=false
 ```
 
+* **Elastic Beanstalk Installation**
+  * In order to add tags to existing Elastic Beanstalk environment, you will need to rebuild the environment with the spot-enabled tag.
+  Follow this guide: http://www.boringgeek.com/add-or-update-tags-on-existing-elastic-beanstalk-environments
+
```

Comment:
I'd say typo: `add tags to an existing`.
Too bad for the external link, I believe hosting that in the project itself could be better, but fine with me.